docs/docs/use-cases.md

```diff
@@ -0,0 +1,26 @@
+# Use Cases
+
+Currently, we provide the following use cases:
+
+## Assisted Prescriptions
+
+This use case shows how to build a custom analytics application with Qlik Core.
+
+* It has a custom-built web UI.
+* The backend uses Qlik Core and serves multiple users with a single document used by Qlik Associative Engine.
+* Load from multiple users is managed by scheduling sessions to different engine instances using a round-robin strategy.
+* The application is deployed on [Amzon Web Services](https://aws.amazon.com/).
+* Docker Swarm is used as container orchestrator.
+
+More info on the Assisted Prescriptions use case can be found [here](./use-cases/assisted-prescriptions.md).
+
+## African Urbanization
+
+This use case shows ... **TODO**
+
+* ...
+* ...
```

Comment:
Automatic scaling of the Qlik Associative Engine depending on the number of active sessions

Reply:
Yeah. This is a disposition PR. Content needs to be added.